### DISCUSSION TITLE
Which http methods are used in Statamic admin?

### DISCUSSION THREAD

**Discussion by imakeitworknet** [statamic/cms#8424] [Q&A] [answered]
Hi. I have an issue with Statamic site moved to a specific shared hosting. The hosting's Apache server limits which http_methods are allowed.

Everything works in Statamic except Saving changes. When I want to **Save** Globals or **save changes in Collection entries**, server responds with **Failed to load resource: the server responded with a status of 405**. It does allow creating new entries, but does not allow updating them.
I talked with hosting owners, and they can allow more http_methods in the server. They just need to know which ones to allow - don't want to allow all.

Anyone knows which http_methods are needed in Statamic control panel?

Thanks!

**Comment by duncanmcclean** [comment, marked as answer]
You'll want to have these enabled: GET / POST / PUT / PATCH / DELETE. I think that's them all.

**Reply by imakeitworknet** [reply]
Thanks, now everything works as it should :)
